src/gwt/src/org/rstudio/core/client/widget/LatchingToolbarButton.java

```diff
@@ -39,10 +49,19 @@ public void setLatched(boolean latched)
       latched_ = latched;
       
       if (latched_)
+      {
+         if (!textIndicatesState_)
+            Roles.getButtonRole().setAriaPressedState(getElement(), PressedValue.TRUE);
          getElement().addClassName(ThemeStyles.INSTANCE.toolbarButtonLatched());
+      }
       else
+      {
+         if (!textIndicatesState_)
+            Roles.getButtonRole().setAriaPressedState(getElement(), PressedValue.FALSE);
```

Comment:
Could be worth pulling these out into a local helper method that's merely `setAriaPressedState(TRUE/FALSE)` and it does the `if` check and the longer incantation (if you ever get tired of copy/pasting all that across all these!)